packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -19,16 +19,20 @@
 }
 
 // The base width of blocks
-.edit-post-visual-editor .block-editor-block-list__block {
-	margin-left: auto;
-	margin-right: auto;
-
+.edit-post-visual-editor .block-editor-block-list__block.has-spacing {
 	@include break-small() {
 		// Compensate for side UI width.
-		.block-editor-block-list__block-edit {
+		> .block-editor-block-list__block-edit {
```

Comment:
Not sure what this does, but it means all full-wide blocks now have margin left and right of them:

<img width="1148" alt="Screenshot 2019-11-01 at 15 58 34" src="https://user-images.githubusercontent.com/1204802/68033811-c1643100-fcc0-11e9-867a-bb781e2144ce.png">